docs/_guide/templates.md

```diff
@@ -497,12 +527,46 @@ render() { return html`<slot name="thing"></slot>`; }
 </my-element>
 ```
 
-## Further reading
-Since LitElement uses lit-html's `html` and `render` functions to render templates you can take advantage
-of the entire lit-html feature-set for writing your templates. You can find further information
-* [on the lit-html homepage](https://lit-html.polymer-project.org)
-* [in the Template Reference](https://lit-html.polymer-project.org/guide/template-reference)
+## Using other lit-html features
+
+Since LitElement uses lit-html's `html` and function to render templates you can take advantage
```

Comment:
s/and function/function

Reply:
good catch. I think that was supposed to be "tag function."